website/blog/2024-01-16-running-tabby-locally-with-rocm.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: running-tabby-locally-with-rocm.md
+title: Running Tabby Locally with AMD ROCm
+authors:
+  - name: Julien Marcuse
+    url: https://github.com/boxbeam
+    image_url: https://github.com/boxbeam.png
+tags: [deployment]
+---
+
+For those using (compatible) AMD graphics cards, you can now run Tabby locally with GPU acceleration using AMD's ROCm toolkit.
+
+Currently, Tabby with ROCm is only supported on Linux, and can only be run directly from a compiled binary. In the future, Tabby will be able to run with ROCm on Windows, and we will distribute a Docker container capable of running with ROCm on any platform.
+
+Before starting, please make sure you are on a supported system and have ROCm installed. The AMD website [details how to install it](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/tutorial/install-overview.html), find the instructions for your given platform.
+
+Once you have installed ROCm, you can [download the precompiled binary for Tabby](https://github.com/TabbyML/tabby/releases/download/nightly/tabby_x86_64-manylinux2014-rocm57) with ROCm, or you can [compile it yourself](https://github.com/TabbyML/tabby/blob/main/CONTRIBUTING.md#local-setup). If compiling yourself, make sure to use the flag `--features rocm` to enable it.
```

Comment:
mention it's currently in nightly build only, but it will be in regular release since 0.8.8